src/effects/VST/VSTEffect.cpp

```diff
@@ -3609,6 +3609,13 @@ VSTEffectValidator::VSTEffectValidator(EffectUIClientInterface& effect,
 VSTEffectValidator::~VSTEffectValidator() = default;
 
 
+VSTEffectInstance::VSTEffectInstance(StatefulPerTrackEffect& effect)
```

Comment:
The commit that stubs `ProcessInitialize` and others is incomplete.  My fault for giving an incomplete description for step 4.

`ProcessFinalize` should also be moved to the instance, and you will know all is correct when the effect also inherits `PerTrackEffect` not `StatefulPerTrackEffect`.

Please figure out step 4 as a whole and squash all the 4.x commits.